communications/rfc.rst

```diff
@@ -0,0 +1,115 @@
+
+In general, DM team members are empowered by the DM Project Manager (PM) and DM Subsystem Scientist (SS) to make decisions on any DM-internal matter --- including technical issues, process improvements, tool choices --- provided those decision meet the criteria of :ldm:`294` C.1 (Empowerment).
+
+For instances where you have a well-formed proposal that does not meet the criteria set out in :ldm:`294` C.1.
+
+.. seealso::
+
+   :doc:`rfd` is an additional process for convening a discussion across DM teams.
+
+.. _rfc-jira:
+
+JIRA workflow for RFCs
+======================
+
+This section describes how DM RFCs are created and resolved.
+
+Create a new issue in JIRA
+--------------------------
+
+Create a new RFC by creating a new JIRA issue:
+
+- Set the **Project** to **Request for Comments**.
+- Set the **Issue Type** to **RFC**.
+- Set the **Component** to **DM**.
+- Give the RFC a descriptive **Summary**.
+- Write the proposal in the issue's **Description** field.
+  Usually it's a good idea to provide some brief background on why this proposal is being made.
+  Ensure that you clearly state your actual, actionable, proposal.
+- Set the **Assignee** to yourself (in typical cases).
+- Set the **Planned End** to at least 72 hours in the future for ordinary issues or 1 week ahead for major issues.
+- Add any relevant team members as **Watchers** (optional).
+
+Once you click **Create**, the RFC will be posted to both the ``dm-devel mailing list`` and the `#dm`_ Slack channel.
+
+RFC discussion and consensus
+----------------------------
+
+Members of DM should comment on the RFC by posting comments to the JIRA issue.
+
+Refer to :ldm:`294` C.2 for guidelines on discussion and consensus building within an RFC.
+
+Adopting an RFC
+---------------
+
+.. _rfc-adopt-it:
+
+Adoption in JIRA
+^^^^^^^^^^^^^^^^
+
+After the **Planned End** date has passed, and there is positive consensus, the Assignee is responsible for adopting the RFC.
+
+Do this by clicking the **Adopt RFC** button on the RFC's JIRA issue page.
+JIRA will pop up a text box where you can confirm the adoption and add an optional text message.
+This message will automatically appear in the `#dm`_ channel.
+
+When you adopt an RFC, ensure that the resolution of the discussion is clearly stated, especially if the resolution is different from the proposal.
+
+.. _rfc-triggering:
+
+Triggering tickets
+^^^^^^^^^^^^^^^^^^
+
+In conjunction with adopting a ticket, create one or more JIRA stories (in the regular DM project) that implement the RFC's decision.
+
+Those stories must have an **Is triggered by** relationship with the parent RFC issue.
+
+Next, see :ref:`rfc-implementing`.
+
+.. _rfc-escalating:
+
+Escalating an RFC
+-----------------
+
+If the discussion is not converging, but you still feel that a decision must be taken, you (or the DMCCB) may escalate the RFC to be considered by the DM Configuration Control Board (DMCCB).
```

Comment:
You don't say how.  "Click the **Escalate RFC** button on the RFC's JIRA issue page."